features/images_appartient_rubrique.feature

```diff
@@ -0,0 +1,22 @@
+#language: fr
+
+Fonctionnalité: Vérifier toutes les images appartiennent bien à une rubrique 
```

Comment:
This feature is unrelated to what you implemented.

Reply:
@sab91